docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst

```diff
@@ -1,12 +0,0 @@
-.. _HTTP Search Requests:
-
-HTTP Search Requests
-====================
-
-Search requests can be also performed by making HTTP requests
-authenticated with the AWS Signature version 4 scheme. See the following
-URLs for more information about AWS V4 authentication.
-
--  http://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html
--  http://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html
-
```

Comment:
Why you are deleting this file in this 82adcfc8d5f4db1aa3224d8473fda6344a8f3fdb commit/PR ?

Reply:
It was there.

Reply:
If you want, I can revert the metadata search mess, but we should definitely review it/put some tickets on the board.

Reply:
> If you want, I can revert the metadata search mess, but we should definitely review it/put some tickets on the board. 
Yes please keep this PR focus on replication. As we discussed this morning, we need to do a big cleanup on the Zenko doc.